charts/k2bridge/values.yaml

```diff
@@ -39,8 +39,8 @@ elasticsearch:
   clusterName: k2bridgees
   masterService:  k2bridgees-master
 
-  image: docker.elastic.co/elasticsearch/elasticsearch-oss
-  imageTag: 7.10.2
+  image: docker.elastic.co/elasticsearch/elasticsearch
+  imageTag: 7.16.2
```

Comment:
Is it perhaps the right time to also update the elastic chart version we use?

Reply:
what's elastic chart?

Reply:
updated.

@AsafMah , its the chart the defines how to deploy elastic. we use it as a dependency
https://github.com/elastic/helm-charts/tree/main/elasticsearch